pyproject.toml

```diff
@@ -12,18 +12,19 @@ multi_line_output = 3
 
 [tool.mypy]
 python_version = "3.12"
-ignore_missing_imports = true
+ignore_missing_imports = false
 disallow_untyped_calls = true
 disallow_untyped_defs = true
 disallow_incomplete_defs = true
 check_untyped_defs = true
-warn_no_return = true
+warn_no_return = false
```

Comment:
Why false

Reply:
Ok, warn_no_return is back =) I'll try to change the function structure to avoid  "missing return statement" by mypy.

Reply:
Not seeing it, still false